client/packages/android/app/src/main/java/org/openmsupply/client/CertWebViewClient.java

```diff
@@ -113,15 +113,14 @@ private boolean validateLocalCertificate(SslCertificate targetCert) {
       */
     private boolean validateNonLocalCertificate(SslCertificate targetCert, NativeApi.omSupplyServer connectedServer) {
         // Calculate SSL fingerprint
-        PublicKey publicKey = get_x509(targetCert).getPublicKey();
         MessageDigest md = null;
         try {
            md = MessageDigest.getInstance("SHA-256");
+           md.update(get_x509(targetCert).getEncoded());
```

Comment:
Awesome! we can merge validateLocalCertificate and validateNonLocalCertificate in the next iteration